recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -34,17 +34,21 @@ set EntryPointPath=%ThisScriptsDirectory%SetPageFileSize.ps1
 if "%SET_PAGEFILE%" NEQ "" (
     if "%CI%" == "azure" (
         REM use different drive than CONDA_BLD_PATH-location for pagefile
-        if "%CONDA_BLD_PATH%" == "C:\\bld\\" (
-            echo CONDA_BLD_PATH=%CONDA_BLD_PATH%; Setting pagefile size to 16GB on D:
-            REM Inspired by:
-            REM https://blog.danskingdom.com/allow-others-to-run-your-powershell-scripts-from-a-batch-file-they-will-love-you-for-it/
-            REM Drive-letter needs to be escaped in quotes
-            PowerShell -NoProfile -ExecutionPolicy Bypass -Command "& '%EntryPointPath%' -MinimumSize 8GB -MaximumSize 16GB -DiskRoot \"D:\""
+        if "%CONDA_BLD_PATH:~0,2%" == "C:" (
```

Comment:
This allows us to check if the drive matches. Variations in the path can then be ignored